module-infrastructure/persistence-db/src/main/kotlin/site/yourevents/common/entity/BaseTimeEntity.kt

```diff
@@ -0,0 +1,21 @@
+package site.yourevents.common.entity
+
+import jakarta.persistence.Column
+import jakarta.persistence.EntityListeners
+import jakarta.persistence.MappedSuperclass
+import org.springframework.data.annotation.CreatedDate
+import org.springframework.data.annotation.LastModifiedDate
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import java.time.LocalDateTime
+
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener::class)
+abstract class BaseTimeEntity(
+    @CreatedDate
+    @Column(nullable = false, updatable = false)
+    var createdAt: LocalDateTime = LocalDateTime.now(),
+
+    @LastModifiedDate
+    @Column(nullable = false)
+    var modifiedAt: LocalDateTime = LocalDateTime.now(),
```

Comment:
LocalDateTime.now() 대신 null로 초기화하고, JPA가 값을 설정하도록 하는 방안이 좋아보입니다!

```suggestion
    @CreatedDate
    @Column(nullable = false, updatable = false)
    var createdAt: LocalDateTime? = null,

    @LastModifiedDate
    @Column(nullable = false)
    var modifiedAt: LocalDateTime? = null,
```

Reply:
`?`에 아직 익숙하지 않아서 미쳐 생각을 못 한 부분이네요!
수정하도록 하겠습니다!